recipes/conda/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "4.0.6" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  fn: conda-{{ version }}.tar.gz
+  url: https://github.com/conda/conda/archive/{{ version }}.tar.gz
+  sha256: 4ca7ec875de13519aeb3eebc9beca23adf3c41e8852c186deb503e8bc2282c40
+
+build:
+  number: 0
+  script: CONDA_DEFAULT_ENV='' python setup.py install
+
+requirements:
+  build:
+    - python
+    - conda-env
+    - pycosat
```

Comment:
I've never built a conda package for conda, frankly. These are certainly run dependencies but if it builds fine without them I see no reason to include them.

One thing about the `conda-env` dependency: please look at our dependencies carefully. It's important that `conda-env` and `conda` remain in rough version lock-step. For instance, `conda 4.1.x` requires `conda-env >=2.5`, and `conda 4.0.x` requires `conda-env <2.5`.

We also use `ruamel_yaml` for `conda 4.1`.